### PR TITLE
A pull request for first code review

### PR DESCRIPTION
**Pull request by dlwub**
Added index.html and index.css files for review.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+}
+/*
+#survey-form{
+      
+}*/
+
+form {
+    display: inline-block;
+    max-width: 50%;
+}
```
Comment (dlwub) [inline index.css:12]: Please use proper indentation (on line 12). Properly indented codes are easier to read.

**Review by dlwub — COMMENTED** [review]
Please make the required changes and request a re-review.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+body {
+    text-align: center;
+}
+/*
+#survey-form{
+      
+}*/
```
[the remark this answers is not in the document]
Comment (1GORDON) [reply index.css:7]: Great review @dlwub 🤝

- [ ] According to [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md), you may please try to be more polite. You may as well use emojis to make your review more human friendly

File: index.css (hunk 1 above)
Comment (1GORDON) [reply index.css:12]: Great review @dlwub 🤝

- [ ] You may please use emojis which make the comments more human-friendly. However, I think this piece of code is actually well indented. You may give a review on something that is more obvious. Maybe commented code or something that actually affects the rendering of a feature.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @dlwub ✋

Good job so far!

## Highlights!
- Good use of the Github Review Changes feature ✔️ 
- Good use of Inline comments  ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] You may please make your reviews more of suggestions than orders.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 1 above)
Comment (dlwub) [inline index.css:12]: There are linter errors (on line 12). May you please avoid them? 👍

**Review by dlwub — COMMENTED** [review]
May you please make the requested changes and ask for re-review? ♻️

File: index.css (hunk 2 above)
Comment (dlwub) [inline index.css:7]: I think it would be better to avoid unused codes left as a comment it reduces the cleanness of the code. 👋

File: index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            </textarea>
+        </div>
+        <br>
+        <button id="submit" type="submit" class="btn btn-primary">Enviar</button>
+
+    </form>
+
+</body>
+
+</html>
```
Comment (dlwub) [inline index.html:88]: There is a linter error (line 88). May you please adjust that? 😄

**Review by dlwub — COMMENTED** [review]
May you please make the requested changes and ask for re-review? ♻️

**Review by mwanawabangona — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @dlwub 👋,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @mwanawabangona in your question so I can receive the notification.

**Pull request merged**